src/functions/process-stream-event.ts

```diff
@@ -13,7 +13,12 @@ import { DynamoDbImage } from '../services/dynamodb-images';
 import { convert } from '../services/entity-conversion';
 import { debugLog } from '../services/logger';
 import { SqlOperation, deriveSqlOperation } from '../services/sql-operations';
-import { transformTechRecord } from '../utils/transform-tech-record';
+import {transformImage, transformTechRecord} from '../utils/transform-tech-record';
```

Comment:
Can you format these imports a little so they are consistent with the existing ones please.